lightly/models/utils.py

```diff
@@ -501,7 +524,7 @@ def random_token_mask(
             Size of the token batch for which to generate masks.
             Should be (batch_size, sequence_length).
         mask_ratio:
-            Percentage of tokens to mask.
+            Proportion of tokens to mask. Default is 0.6 .
```

Comment:
```suggestion
            Proportion of tokens to mask.
```

The default value should not be documented as it is already part of the function definition:
```
def random_token_mask(
...
    mask_ratio: float = 0.6,
...
):
```

Also documenting it in the docstring creates a lot of overhead because it is difficult to verify that the actual default value and the documented default are always the same. This is especially a problem when code gets updated and changes to the docstrings get forgotten.

Could you also remove the default values in the docstrings in the other functions?

Reply:
@guarin done